Automattic/wp-calypso
# Add outline to card links

#### Changes proposed in this Pull Request

* Add an outline when a card link is in focus. This improves accessibility and keyboard navigation.

#### Testing instructions

* Go to `/people` and select any from the navigation panel.
* Try navigating the section with your keyboard (use `tab`).
* You can't see when a card item is in focus.

Fixes #31634

#### After

* Added the outline back to the card item.
![2019-04-04 15-44-54 2019-04-04 15_45_47](https://user-images.githubusercontent.com/3276087/55591076-df00f600-56f1-11e9-9c7e-4090ace05032.gif)
